api/python/t4/api.py

```diff
@@ -82,22 +94,28 @@ def get(src, version=None):
     return deserialize_obj(data, target), meta.get('user_meta')
 
 
-def delete(path):
-    """Delete an object from T4.
-
-    Does not delete local files.
+def delete(target):
+    """Delete an object.
 
     Parameters:
-        path (str): Path of object to delete
+        target (str): URI of the object to delete
```

Comment:
This is somewhat opposed to what Aneesh wants, he wants to remove as much as possible "s3:/" from the API users, we should discuss.